dazel.py

```diff
@@ -146,27 +150,51 @@ def from_config(cls):
 
     def send_command(self, args):
         term_size = shutil.get_terminal_size()
-        command = "%s exec -i -e COLUMNS=%s -e LINES=%s -e TERM=%s %s %s %s %s %s %s %s %s" % (
+
+        docker_exec_command = "%s exec -i -e COLUMNS=%s -e LINES=%s -e TERM=%s %s %s %s %s" % (
             self.docker_command,
             term_size.columns, term_size.lines,
             os.environ.get("TERM", ""),
             "-t" if sys.stdout.isatty() else "",
             "--privileged" if self.docker_run_privileged else "",
             ("--user=%s" % self.user
              if self.user else ""),
-            self.instance_name,
+            self.instance_name)
+
+        if not self.user:
+            output_args = ("--output_user_root=%s --output_base=%s" % (
+                TEMP_BAZEL_OUTPUT_USER_ROOT, self.bazel_output_base)
+            if self.command and self.bazel_output_base else "")
+        else:
+            output_args = ("--output_user_root=%s" % (self.bazel_user_output_root)
+            if self.command and self.bazel_user_output_root else "")
+
+        command = "%s %s %s %s %s" % (
```

Comment:
you can leave it as `command` instead of `docker_exec_command` above, and have `command +=` here.

In faact, it'll probably be better to do the `if not self.user:` thingy before the command, and then leave the command as one large string format as it was before.

Reply:
I cannot do that since I am using docker_exec_command further down to fix the symbolic paths on windows

Reply:
Got it.